src/test/java/Entities/PlayerTest.java

```diff
@@ -128,11 +128,11 @@ public void testBuildHouseSuccess() {
     public void testBuildHouseHotel() {
```

Comment:
The buildHouse method is also moved into the PlayerLogic class so we may need to substitute this test with that in playerLogic test.

Reply:
I agree, we're keeping this here for now just in case there's any changes to the implementations. But if not, this test will be removed on Monday